cypress/integration/database-page.spec.js

```diff
@@ -0,0 +1,181 @@
+const constants = {
+    rootUrl: 'https://www.bcpl.info/books-and-more/databases.html',
+    selectors: {
+        databaseListContainer: '.database-cards',
+        databaseListItems: '.card:not(.ng-hide)',
+        filters: '#filters',
+        filterButton: '[data-toggle="collapse"]'
+    }
+};
+
+describe('Database Page', () => {
+    /** Helper Functions */
+    const assertNumberOfResearchDatabases = () => {
+        getReasearchDatabases()
+            .then(researchDatabases => {
+                numberOfResearchDatabases = researchDatabases.length;
+                expect(numberOfResearchDatabases).to.be.greaterThan(0);
+            });
+    };
+
+    const getReasearchDatabases = () => 
+        cy
+            .wait(500) // wait for ajax to complete, yes i know this isn't great
```

Comment:
As discussed we will need to replace this with some condition other than time in the future.  Maybe there's an element that's expected to appear or change after the AJAX call that can be watched for.